README.md

```diff
@@ -134,32 +140,32 @@ These numbers are based on [scalability tests](https://github.com/kubernetes/kub
 
 Note that if CPU limits are set too low, kube-state-metrics' internal queues will not be able to be worked off quickly enough, resulting in increased memory consumption as the queue length grows. If you experience problems resulting from high memory allocation, try increasing the CPU limits.
 
-### kube-state-metrics vs. Heapster
+### kube-state-metrics vs. [Heapster(metrics-server)](https://github.com/kubernetes-incubator/metrics-server)
```

Comment:
It should be: Heapster([metrics-server](https://github.com/kubernetes-incubator/metrics-server)).
I.e., the link is only related with metrics-server.